### PR TITLE
BaSP-TG-119: Employee timesheets page

### PR DESCRIPTION
**Pull request by MaicolFreire001**
Created a view in which you have a list with all the timesheets that the employee is part of and having the option to add timesheets

Work by Cristian Gerster and Maicol Freire
‌

### CODE REVIEW

File: src/Components/Pages/Employee/TimeSheetsList/Form/index.jsx [hunk 1]
```diff
@@ -0,0 +1,144 @@
...
+    dispatch(postTimeSheet(data));
+    setTypeModal('Success');
+    setTextModal('TimeSheet added successfully');
+    openModal();
+  };
+
+  return (
+    <div className={styles.container}>
+      <h2 className={styles.title}>Add new time sheet</h2>
+      <Button href="/time-sheets" style="roundedSecondary" disabled={false} text="X" />
```
Comment (JuliMonte5) [inline src/Components/Pages/Employee/TimeSheetsList/Form/index.jsx:76]: change href to "timesheets" which is used now in the layout

File: src/Components/Pages/Employee/TimeSheetsList/Form/index.jsx [hunk 2]
```diff
@@ -0,0 +1,144 @@
...
+        style="squaredPrimary"
+        disabled={false}
+        text="Save"
+      />
+      <MessageModal
+        type={typeModal}
+        isOpen={showModal}
+        message={textModal}
+        handleClose={closeModal}
+        goBack={'/time-sheets'}
```
Comment (JuliMonte5) [inline src/Components/Pages/Employee/TimeSheetsList/Form/index.jsx:138]: same as 76

File: src/Components/Pages/Employee/TimeSheetsList/time-sheets.module.css [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+  margin-right: 75px;
+  width: 80%;
+  display: flex;
+  align-items: flex-end;
+  flex-direction: column;
+  flex-wrap: wrap;
+  align-content: center;
+}
+
+h2 {
```
Comment (JuliMonte5) [inline src/Components/Pages/Employee/TimeSheetsList/time-sheets.module.css:12]: should be a class

File: src/Components/Pages/Employee/TimeSheetsList/Form/form.module.css [hunk 4]
```diff
@@ -0,0 +1,81 @@
+.title{
```
[the remark this answers is not in the document]
Comment (MaicolFreire001) [reply src/Components/Pages/Employee/TimeSheetsList/Form/form.module.css:1]: Done

File: src/Components/Pages/Employee/TimeSheetsList/Form/form.module.css [hunk 5]
```diff
@@ -0,0 +1,81 @@
...
+
+.textareacontainer {
+    display: flex;
+    flex-direction: column;
+    width: 40%;
+    height: fit-content;
+    justify-content: flex-start;
+}
+
+textarea {
```
[the remark this answers is not in the document]
Comment (MaicolFreire001) [reply src/Components/Pages/Employee/TimeSheetsList/Form/form.module.css:64]: Done

File: src/Components/Pages/Employee/TimeSheetsList/Form/index.jsx (hunk 1 above)
Comment (MaicolFreire001) [reply src/Components/Pages/Employee/TimeSheetsList/Form/index.jsx:76]: Done

File: src/Components/Pages/Employee/TimeSheetsList/Form/index.jsx (hunk 2 above)
Comment (MaicolFreire001) [reply src/Components/Pages/Employee/TimeSheetsList/Form/index.jsx:138]: Done

File: src/Components/Pages/Employee/TimeSheetsList/time-sheets.module.css (hunk 3 above)
Comment (MaicolFreire001) [reply src/Components/Pages/Employee/TimeSheetsList/time-sheets.module.css:12]: Not used anymore, removed altogether

Comment (MaicolFreire001) [reply src/Components/Pages/Employee/TimeSheetsList/time-sheets.module.css:12]: Done

File: src/Components/Pages/Employee/TimeSheetsList/Form/index.jsx (hunk 1 above)
Comment (JuliMonte5) [reply src/Components/Pages/Employee/TimeSheetsList/Form/index.jsx]: Still doesn't redirect to the employee timesheets, should be "/employee/timesheets". Check it after the change

**Review by Ignacio-Sormani — APPROVED** [review]

**Review by GonzaloAsencio — APPROVED** [review]

**Review by fede230298 — APPROVED** [review]

**Pull request merged**